### ISSUE TITLE
LayoutInterfaceTest tests are failing.

### ISSUE THREAD

**Issue by Gormartsen** [backdrop/backdrop-issues#1865]
Issue appear between PR 1373 and 1375.

Possible source is this commit (not confirmed, need more tests for each commit):
https://github.com/backdrop/backdrop/commit/4d1052ed2f1837564c67fca5e5ce6901c58af176

Both php53 and php7 failed with the same report:
```simpletest

---- Layout: Layout Interface Test (LayoutInterfaceTest) ----
Status    Group      Filename          Line Function                            
--------------------------------------------------------------------------------
Fail      Other      layout.test        489 LayoutInterfaceTest->testBlockEntit
    Failed to set field entity_id to 1
Fail      Other      layout.test        489 LayoutInterfaceTest->testBlockEntit
    Found the requested form fields at
Fail      Other      layout.test        490 LayoutInterfaceTest->testBlockEntit
    Found the Save configuration button
Fail      Other      layout.test        490 LayoutInterfaceTest->testBlockEntit
    Found the requested form fields at
Fail      Other      layout.test        491 LayoutInterfaceTest->testBlockEntit
    Found the Save layout button
Fail      Other      layout.test        491 LayoutInterfaceTest->testBlockEntit
    Found the requested form fields at
Fail      Other      layout.test        494 LayoutInterfaceTest->testBlockEntit
    Block shown and node ID matches.
Fail      Other      layout.test        514 LayoutInterfaceTest->testBlockEntit
    Failed to set field entity_id to 3
Fail      Other      layout.test        514 LayoutInterfaceTest->testBlockEntit
    Found the requested form fields at
Fail      Other      layout.test        515 LayoutInterfaceTest->testBlockEntit
    Found the Save configuration button
Fail      Other      layout.test        515 LayoutInterfaceTest->testBlockEntit
    Found the requested form fields at
Fail      Other      layout.test        516 LayoutInterfaceTest->testBlockEntit
    Found the Save layout button
Fail      Other      layout.test        516 LayoutInterfaceTest->testBlockEntit
    Found the requested form fields at
Fail      Other      layout.test        528 LayoutInterfaceTest->testBlockEntit
    Block shown when user ID and node ID matches.
Fail      Other      layout.test       1201 LayoutInterfaceTest->testTaxonomyCo
    Failed to set field bundles[tags] to 1
Fail      Other      layout.test       1201 LayoutInterfaceTest->testTaxonomyCo
    Found the requested form fields at
Fail      Other      layout.test       1202 LayoutInterfaceTest->testTaxonomyCo
    Found the Create layout button
Fail      Other      layout.test       1202 LayoutInterfaceTest->testTaxonomyCo
    Found the requested form fields at
Fail      Other      layout.test       1204 LayoutInterfaceTest->testTaxonomyCo
    "Layout created. Blocks may now be added to this layout." found

```

https://travis-ci.org/backdrop/backdrop/jobs/130288367
https://travis-ci.org/backdrop/backdrop/jobs/130288364

**Comment by Gormartsen** [comment]
relate to #1020

**Comment by quicksketch** [comment]
Thanks for making this issue @Gormartsen! I've filed a PR to fix this at https://github.com/backdrop/backdrop/pull/1376.

**Comment by Gormartsen** [comment]
Fixed in backdrop/backdrop#1376
